molecule/default/molecule.yml

```diff
@@ -19,6 +19,8 @@ provisioner:
   name: ansible
   lint:
     name: ansible-lint
+    options:
```

Comment:
This should no longer be necessary, correct? I can not find any skip tag in the changes.

Reply:
Correct, will be deleted.